docs/source/envs.rst

```diff
@@ -4,120 +4,133 @@
       - FALSE
     * - .. envvar:: RADICAL_PILOT_STRICT_CANCEL
-      - Limit task cancelation, not to force state "CANCELLED" on a Task Manager side. Note that corresponding command for pilot(s) to cancel tasks is published anyway
+      - Limits task cancelation by not forcing the state "CANCELLED" on the Task Manager.
       - {NOT_SET}
     * - .. envvar:: RADICAL_DEFAULT_LOG_TGT
-      - The log target
+      - Log target
       - .
     * - .. envvar:: RADICAL_DEFAULT_LOG_DIR
-      - The Log dir
+      - Log directory
       - $PWD
     * - .. envvar:: RADICAL_DEFAULT_LOG_LVL
-      - The default log level when not set
+      - The default log level when not explicitly set
       - ERROR
     * - .. envvar:: RADICAL_DEFAULT_REPORT
       - Flag to turn reporting on [TRUE/1] or off [FALSE/0/OFF]
       - TRUE
     * - .. envvar:: RADICAL_DEFAULT_REPORT_TGT
-      - Similar to ..LOG_TGT: List of comma separated targets [0/null, 1/stdout, 2/stderr, ./{report_name/path}] to write the report info
+      - List of comma separated targets [0/null, 1/stdout, 2/stderr, ./{report_name/path}]
       - stderr
     * - .. envvar:: RADICAL_DEFAULT_REPORT_DIR
-      - Directory of Reporter module
+      - Directory used by the reporter module
       - $PWD
     * - .. envvar:: RADICAL_DEFAULT_PROFILE
       - Flag to turn profiling/tracing on [TRUE/1] or off [FALSE/0/OFF]
       - TRUE
     * - .. envvar:: RADICAL_DEFAULT_PROFILE_DIR
-      - Directory for profiles/traces
+      - Directory where to store profiles/traces
       - $PWD
 
-.. raw:: html
+Logger
+------
 
-   <hr>
+`ru.Logger` instances have a name and a name space.
 
-.. list-table:: Logging-related Env vars.
-    `ru.Logger` instances have a name and a name space.
-    :widths: 35 100 20
+.. list-table::
+    :widths: auto
     :header-rows: 1
 
-    * - Environment Variable Name
+    * - Name
       - Description
       - Default value
-    * - <NS>_LOG_LVL
-      - Logging level ["DEBUG", "INFO", "WARNING", "ERROR", "CRITICAL"] to control the debug output for a corresponding namespace NS, where NS can be applied as for a specific package (e.g., "RADICAL_PILOT_LOG_LVL" or "RADICAL_UTILS_LOG_LVL") or for a whole stack (e.g., "RADICAL_LOG_LVL")
+    * - .. envvar:: <NS>_LOG_LVL
+      - Logging level ["DEBUG", "INFO", "WARNING", "ERROR", "CRITICAL"]
       - Refer `RADICAL_DEFAULT_*`
-    * - <NS>_LOG_TGT
+    * - .. envvar:: <NS>_LOG_TGT
       - Used for the log targets
       - Refer `RADICAL_DEFAULT_*`
 
-.. note:: The name space is used to derive environmental variable names for log levels and targets. If no ns is given, the ns is derived from the name. Eg. the name `radical.pilot` becomes `RADICAL_PILOT`.
+.. note:: The name space is used to derive environmental variable names for log levels and targets. If no name space is given, it is derived from the variable name. For example, the name `radical.pilot` becomes `RADICAL_PILOT`.
```

Comment:
```
`radical.pilot`
```
add double quotes since it is rst-file -> ` ``radical.pilot`` `, and the same for `RADICAL_PILOT`

Reply:
Done.